modules.d/95nfs/nfs-lib.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
-. /lib/net-lib.sh
```

Comment:
Why did you remove this?

https://github.com/dracutdevs/dracut/blob/52351cfa049a9594d539e6a5337d591e8039ab80/modules.d/95nfs/nfs-lib.sh#L28